manifests/virtualhost.pp

```diff
@@ -8,6 +8,7 @@
   Optional[String]               $user           = undef,
   Optional[String]               $group          = undef,
   Hash                           $components     = {},
+  Optional[Array[String]]        $disco_items    = undef,
```

Comment:
```suggestion
  Array[String[1]]               $disco_items    = [],
```

Reply:
solved in 91da1e311bb6e945937bca205d38e033635f574e